integration/run_test.go

```diff
@@ -442,7 +460,7 @@ func TestKritisPods(t *testing.T) {
 				t.Fatalf("deployment should have failed but succeeded\n%s",
 					getKritisLogs(t))
 			}
-
+			checkAndDeleteAttestations(t, testCase.attestedImages)
```

Comment:
I don't understand what this is testing at all. Can you refactor this so that the comparisons are done within the test function and not in a helper?

"Readers should need to read only the test body to understand the test". 

See also: 

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures
https://g3doc.corp.google.com/go/g3doc/go-readability-review-comments.md?cl=head#TOC-Assert

Reply:
sure. will do.